plutus-core/src/Language/PlutusCore/Core/Variables.hs

```diff
@@ -0,0 +1,137 @@
+module Language.PlutusCore.Core.Variables
+    ( fvTerm
+    , ftvTerm
+    , ftvTy
```

Comment:
Moved that stuff to a separate file within `PlutusCore/Core`, because now `Instances.Eq` depends on it.